paper-dialog-common.css

```diff
@@ -9,15 +9,15 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
 */
 
 :host {
-  display: block;
-  margin: 24px 40px;
-  -webkit-overflow-scrolling: touch;
+  @apply(--paper-font-body1);
+  @apply(--shadow-elevation-16dp);
 
+  -webkit-overflow-scrolling: touch;
   background: var(--paper-dialog-background-color, --primary-background-color);
   color: var(--paper-dialog-color, --primary-text-color);
+  display: block;
+  margin: 24px 40px;
 
-  @apply(--paper-font-body1);
```

Comment:
i'm just worried that they'll have side effects that we're not testing. Order of css statements matters, and we don't have a standard about where these apply should actually go (that they go at the top is just anecdotal; clearly here they don't). I can't 100% guarantee without looking at each of these applies that they don't set (for example) a `display: relative` that the other `display: block` is now overriding.

I know this sounds annoying but it's a case of "if it ain't broke don't fix it"... :(